app/models/manageiq/providers/ovirt/infra_manager/iso_datastore.rb

```diff
@@ -0,0 +1,54 @@
+class ManageIQ::Providers::Ovirt::InfraManager::IsoDatastore < ManageIQ::Providers::Ovirt::InfraManager::Storage
+  supports :iso_datastore
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id
+
+  virtual_delegate :name, :to => :ext_management_system, :allow_nil => true, :type => :string
+
+  # Synchronize advertised images as a queued task. The
+  # queue name and the queue zone are derived from the EMS.
+  #
+  def synchronize_advertised_images_queue
+    MiqQueue.put_unless_exists(
+      :class_name  => self.class.name,
+      :instance_id => id,
+      :method_name => "synchronize_advertised_images",
+      :queue_name  => ext_management_system.queue_name_for_ems_operations,
+      :zone        => ext_management_system.try(:my_zone),
+      :role        => "ems_operations"
+    )
+  end
```

Comment:
If we're getting these in refresh below do we need this still?

Reply:
No these are no longer needed, thanks